Dockerfile

```diff
@@ -1,28 +1,36 @@
-FROM golang:1.10.1-alpine3.7
+FROM golang:1.10.4-alpine3.7
 
 LABEL maintainer="Minio Inc <dev@minio.io>"
 
 ENV GOPATH /go
-ENV PATH $PATH:$GOPATH/bin
 ENV CGO_ENABLED 0
+
+WORKDIR /go/src/github.com/minio/
+
+RUN  \
+     apk add --no-cache ca-certificates 'curl>7.61.0' git && \
```

Comment:
This line is not necessary

Reply:
You only need to install `git`. The image has certs. 

> curl might not be needed.

:+1:

Reply:
Yeah that is pretty pathetic, go needs `git` and Golang image doesn't have it. That is why we kept everything that we need :-)